StortSpelProjekt/Game/src/Gamefiles/Components/HealthComponent.h

```diff
@@ -0,0 +1,43 @@
+#ifndef HEALTH_COMPONENT_H
+#define HEALTH_COMPONENT_H
+
+#include "..\..\Engine\src\ECS\Components\Component.h"
+#include "..\Events\Events.h"
```

Comment:
change \ to / for consistency with rest of the project

Reply:
done